params/protocol_params.go

```diff
@@ -138,6 +138,7 @@ const (
 	FractionMulExpGas           uint64 = 50     // Cost of performing multiplication and exponentiation of fractions to an exponent of up to 10^3.
 	ProofOfPossessionGas        uint64 = 350000 // Cost of verifying a BLS proof of possession.
 	GetValidatorGas             uint64 = 1000   // Cost of reading a validator's address.
+	GetValidatorBLSGas          uint64 = 1000   // Cost of reading a validator's BLS public key.
```

Comment:
how did we obtained this value?? cc  @prestwich @kobigurk

Reply:
@mrsmkl's reasoning was that since it's cached, the cost is the same as `GetValidatorGas`, because it's just reading a value from a similar place.

Reply:
The benchmarking code is in this repo: https://github.com/mrsmkl/b12-sol/